src/mappings/metadataHelpers.ts

```diff
@@ -4,10 +4,23 @@ import { jsonToString } from './utils'
 import { createOrLoadSubgraphCategory, createOrLoadSubgraphCategoryRelation, createOrLoadNetwork } from './helpers'
 
 export function fetchGraphAccountMetadata(graphAccount: GraphAccount, ipfsHash: string): void {
+  let ipfsData = ipfs.cat(ipfsHash)
```

Comment:
Hmm, i remember I did a commit where i made this file git ignored. in fact, `.gitignore` contains it right now. 

i think we can just delete this file from the PR

Reply:
Yeah, I'm not 100% sure why it was still being tracked, but since the old version contained conflict strings, I made sure to at least remove that.
I'm gonna go ahead and delete them on the update.

Reply:
yeah i think gitignore does a weird thing where if the file EXISTS it doesnt catch it, and will continue to allow you to edit it. but then once you delete it, gitignore works properly.